zetaclient/common/utils.go

```diff
@@ -50,3 +53,13 @@ func PrintComplianceLog(
 	logWithFields1.Warn().Msg(logMsg)
 	logWithFields2.Warn().Msg(logMsg)
 }
+
+// GasPriceMultiplier returns the gas price multiplier for the given chain
+func GasPriceMultiplier(chainID int64) (float64, error) {
```

Comment:
I didn't see this file but common/utils is very generic. Both names are generally not advised for package name.
Can we create new packages?
Compliance in a compliance package. 
This GasPriceMultiplier maybe in zetabridge since this is for posting the gas price on zetachain

Reply:
https://github.com/zeta-chain/node/pull/1881/commits/b1460f6180f10962c0c7d075811f879a4bbfef50